docs/MigrationGuide.md

```diff
@@ -42,6 +42,15 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 * [#4745](https://github.com/TouK/nussknacker/pull/4745) Change `api/properties/*/validation` endpoint request type
   * Replace `processProperties` with `additionalFields`
   * Add `id` field for scenario or fragment id
+* [#5039](https://github.com/TouK/nussknacker/pull/5039) Changes in endpoints 
+  * `api/parameters/*/suggestions` request
+    * Add `scenarioName` field 
+    * Add `processProperties` field 
+    * `variables` field should have only local variables now
```

Comment:
Maybe rename it to `localVariables` then?

Reply:
After rethinking, I'd rather change it to `variableTypes` like it already is in `ParametersValidationRequest` and `NodeValidationRequest` to make it concise and more clear that it contains only types, not real variables. I though about `localVariableTypes` in every place, but:
1. It is a lot of places, we also return variables types in responses and everwhere they are local
2. Names length will be increased significantly e.g. we should rename `branchVariableTypes` to `branchLocalVariableTypes` to be concise 
3. Currently client side has no access to global variables in any place, global variables become a hidden, internal detail of BE so from client perspective, local variables are the only variables that it knows.